ec2_restart_instance_by_filter_and_az/ec2_restart_instance_by_filter_and_az.json

```diff
@@ -0,0 +1,63 @@
+{
+    "version": "1.0.0",
+    "title": "My application is resilient to an ec2 instance that is restarted by applying a filter for a named region",
+    "description": "Can my application maintain its minimum resources in the event of a filtered ec2 instance restarting? The indstance is selected by filter in a named region",
+    "tags": [
+        "platform:aws",
+        "service:ec2"
+    ],
+    "configuration": {
+        "service_url": {
+            "type": "env",
+            "key": "APPLICATION_ENTRYPOINT_URL"
+        },
+        "instanceid": {
+            "type": "env",
+            "key": "INSTANCE_ID"
+        },
+        "availability_region": {
+            "type": "env",
+            "key": "AVAILABILITY_REGION"
+        }
+    },
+    "contributions": {
+        "availability": "high",
+        "reliability": "high",
+        "safety": "none",
+        "security": "none",
+        "performability": "medium"
+    },
+    "steady-state-hypothesis": {
+        "title": "Application is normal",
+        "probes": [
+            {
+                "type": "probe",
+                "name": "application-must-respond-normally",
+                "tolerance": 200,
+                "provider": {
+                    "type": "http",
+                    "url": "${service_url}",
+                    "timeout": 3
+                }
+            }
+        ]
+    },
+    "method": [
+        {
+            "type": "action",
+            "name": "stop_instance",
+            "provider": {
+                "type": "python",
+                "module": "chaosaws.ec2.actions",
+                "func": "stop_instance",
+                "arguments": {
+                    "az": "${availability_region}",
```

Comment:
"az" is usually taken to mean "availability zone", but we seem to have gone for "region" which is unfamiliar to me. I could be wrong though?